### PR TITLE
lutris: update to 0.4.16_2.

### PR DESCRIPTION
**Pull request by nonchip**
this patch fixes an issue with people using `LD_LIBRARY_PATH` in their lutris system config.

also, after talking to @chneukirchen we realized the original maintainer seems very absent and decided for me to take care of this package since I use it on an almost daily basis and am at least a bit able to figure out my way in the codebase and should be able to test and/or debug PRs.

### CODE REVIEW

File: srcpkgs/lutris/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'lutris'
 pkgname=lutris
 version=0.4.16
-revision=1
+revision=2
 noarch=yes
 build_style=python3-module
 pycompile_module="${pkgname}"
```
Comment (cr6git) [inline srcpkgs/lutris/template]: Use `lutris` instead of `${pkgname}` you will save four characters ;)

Comment (nonchip) [reply srcpkgs/lutris/template]: i didn't touch that :D
and actually it's nicely modular

**Comment by asergi** [comment]
`0.4.17` is out.

**Comment by asergi** [comment]
Superseded by #14608.

**Pull request closed without merging**